Tasks/FtpUploadV2/task.loc.json

```diff
@@ -160,7 +151,7 @@
     }
   ],
   "execution": {
-    "Node": {
+    "Node10": {
```

Comment:
Node 10?  The agents run Node 6

Reply:
The ftp library requires node v8 or above.  We pack a node10 handler in the latest agent.